tests/tests.py

```diff
@@ -1692,9 +1694,9 @@ def test_create_version(self):
 
     def test_create_version_with_project_obj(self):
         project = self.jira.project(self.project_b)
-        version = self.jira.create_version('new version 1', project,
+        version = self.jira.create_version('new version 2', project,
```

Comment:
Are we running tests in parallel?! (If so, where is this configured?)

Specifically, I know we run multiple Travis builds concurrently, but _within a single Travis build_ (or local test run, for that matter) do multiple tests run in parallel?

This change, and the setup of the `JiraTestManager` imply that we do, but I can't see any evidence of this in the configuration or Travis test run logs, so I'm a bit confused.